SimG4Core/Geometry/interface/SensitiveDetectorCatalog.h

```diff
@@ -3,20 +3,23 @@
 
 #include <map>
 #include <string>
+#include <string_view>
 #include <vector>
+#include <unordered_set>
 
 class SensitiveDetectorCatalog {
 public:
-  using MapType = std::map<std::string, std::vector<std::string>>;
+  using MapType = std::map<std::string, std::unordered_set<std::string>>;
```

Comment:
@ianna @civanch I assume that this move is induced by the fact that the explicit order of the strings is irrelevant, am I correct?

Reply:
@fabiocos - as far as I understand from its usage, yes